Model/SofortPaymentMethod.php

```diff
@@ -167,4 +171,16 @@ public function getPublicFieldNames()
             ['additionalInfo', 'countryCode', 'logoType', 'deleteCanceledOrder', 'deleteFailedOrder']
```

Comment:
We discussed that 'deleteCanceledOrder', 'deleteFailedOrder' will be moved to the abstract method so we save code in all payment method classes.

Reply:
we cannot move it, because for sepa ct it makes no sense to delete the order if the transaction fails